oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -30,82 +30,64 @@ OEP-14: Archiving edX GitHub Repositories
 
-1. Post a notice to `Open edX Deprecation Announcements`_ announcing that the repository
-   will be archived, and inquiring if anyone would like to take ownership of
-   the repo. If there are no responses after 2 work days, skip to `Archive
-   Steps`_.
+1. Post a notice to `Open edX Deprecation Announcements`_ announcing that the repository will be archived, and inquiring if anyone would like to take ownership of the repo. If there are no responses after 2 work days, skip to `Archive Steps`_.
 
-2. If someone does wish to take ownership of the repository, email the internal
-   edX developers mailing list to see if there are any objections. If there are
-   no objections after 2 work days, post to `Open edX Deprecation Announcements`_ that the
-   transfer will take place.
+2. If someone does wish to take ownership of the repository, email the internal edX developers mailing list to see if there are any objections. If there are no objections after 2 work days, post to `Open edX Deprecation Announcements`_ that the transfer will take place.
 
-3. Create an IT help ticket for the repository to be transferred to the new
-   organization.
+3. Create an IT help ticket for the repository to be transferred to the new organization.
 
-4. Once the transfer has occurred, create a fork of the transferred repository
-   into the `edx organization`_ and follow the `Archive Steps`_ below for the
-   forked repo.
+4. Once the transfer has occurred, create a fork of the transferred repository into the `edx organization`_ and follow the `Archive Steps`_ below for the forked repo.
 
 .. _Open edX Deprecation Announcements: https://discuss.openedx.org/c/announcements/deprecation
 
 
 Archive Steps
 -------------
 
-1. Update the README.rst file in the repository to state that it is archived,
-   using the `README Archive Statement`_ below.
+These steps should be followed for all repos within the edX organization(forks included). After some experiments with keeping archived repos in the ``edx`` organization, we've learned that having abandoned code show up in searches hinders work to understand the current state of the system and the risk around new work, particularly deprecations and API changes. Thus we decided to move all archived repositories to a separate org.
+
+1. Update the README.rst file in the repository to state that it is archived, using the `README Archive Statement`_ below.
 
 2. Update the openedx.yaml file, creating it if necessary:
 
    - Add ``archived: True``.
 
    - Remove the ``openedx-release`` key if it is present.
 
-   - It is not necessary for the openedx.yaml file to define an owner
-     for archived repos.
+   - It is not necessary for the openedx.yaml file to define an owner for archived repos.
+
+3. Unless you have the relevant permissions to the work in this step, create an IT help ticket and ask them to do the following:
```

Comment:
General question -- this "open edX proposal" seems very edx.org specific in working with our internal IT department in a few different places (which it appears to have done before), does the semantics of this still make sense?

Reply:
There are many  Open edX related process right now that have ties to edX Inc process and component teams, I think even if not everyone in the community could execute the process, it's still worth sharing the process out.  I think it's okay to leave as is now.